### PR TITLE
Updates doc links to opencrux.com

### PR DESCRIPTION
**Pull request by naomijub**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,16 +2,17 @@
...
+- [x] Via [`HTTP`](https://opencrux.com/reference/http.html#start-http-server) using the [`HTTP API`](https://opencrux.com/reference/http.html#http-api).
 - [x] Async support.
 - [ ] Clojure.api. (To be evaluated.)
 - [ ] FFI. (To be evaluated.)
 
 > Other solutions may be added after the first release.
 
-* For information on Crux and how to use it, please follow the link to [opencrux](https://opencrux.com/docs#restapi).
-* [**Crux FAQ**](https://opencrux.com/docs#faqs)
+* For information on Crux and how to use it, please follow the link to [opencrux](https://opencrux.com/reference/http.html#http-api).
```
Comment (otaviopace) [inline README.md:13]: Should this link really point to the HTTP part? Or should it be more generic? :thinking:

Comment (naomijub) [reply README.md]: removed as it already has the getting started 69641e5

File: README.md [hunk 2]
```diff
@@ -61,7 +62,8 @@ let client = Crux::new("127.0.0.1","3000").http_client();
 
 ## Http Client
 Once you have called `http_client` you will have an instance of the `HttpClient` struct which has a bunch of functions to query Crux on Docker and Standalone HTTP Server:
-* [`state`](https://docs.rs/transistor/1.3.2/transistor/http/struct.HttpClient.html#method.state) queries endpoint [`/`](https://opencrux.com/docs#rest-home) with a `GET`. No args. Returns various details about the state of the database.
+
+* [`state`](https://docs.rs/transistor/1.3.3/transistor/http/struct.HttpClient.html#method.state) queries endpoint [`/`](https://opencrux.com/reference/http.html#home) with a `GET`. No args. Returns various details about the state of the database.
```
Comment (otaviopace) [inline README.md:65]: (https://docs.rs/transistor/1.3.3/transistor/http/struct.HttpClient.html#method.state) The resource doesn't exist

Comment (naomijub) [reply README.md:65]: version not released

**Pull request merged**